modules/blogging/src/Volo.Blogging.Application.Contracts/Volo/Blogging/BloggingPermissions.cs

```diff
@@ -36,30 +36,5 @@ public static class Comments
             public const string Update = Default + ".Update";
             public const string Create = Default + ".Create";
         }
-
-        public static string[] GetAll()
```

Comment:
Sorry for the misunderstanding. I didn't mean to remove GetAll() methods, but use ReflectionHelper inside these methods. 
It will be a unified model (where GetAll may want to manually or in some other way implement it).

Reply:
Got it. 😅